.codeclimate.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
https://github.com/ManageIQ/manageiq/blob/master/lib/generators/provider/templates/.codeclimate.yml is the copy that I would recommend copying from.  I cleaned up the one in manageiq-gems-pending in https://github.com/ManageIQ/manageiq-gems-pending/pull/254